pkg/resource/queue/sdk.go

```diff
@@ -248,6 +254,14 @@ func (rm *resourceManager) sdkUpdate(
 	defer func() {
 		exit(err)
 	}()
+	if delta.DifferentAt("Spec.Tags") {
+		if err := rm.syncTags(ctx, desired, latest); err != nil {
+			return nil, err
+		}
+	}
+	if !delta.DifferentExcept("Spec.Tags") {
+		return desired, nil
+	}
```

Comment:
Now i'm wondering... should we just start generating this kind of stuff + the hooks.go ? literrally the only difference with other services is the name of the API call that adds/removes/lists the tags